komodo/cli.py

```diff
@@ -103,10 +103,14 @@ def _main(args):
         _print_timings(timings[-1])
 
         shell(f"mv {args.release + str(tmp_prefix)} {args.release}")
-        shell(
-            "rmdir -p --ignore-fail-on-non-empty "
-            f"{args.release + str(tmp_prefix.parent)}"
-        )
+        if sys.platform == "darwin":
+            # OS X
+            shell("rm -rf " f"{args.release + str(tmp_prefix.parent)}")
```

Comment:
This would introduce different behaviour on OS X, as this will delete it always, while the rmdir version will let the directory stay there if it is nonempty.

Reply:
(or am I missing something?)

Reply:
Maybe `os.removedirs()` can be used instead for portability.

Reply:
You are correct, --ignore-fail-on-non-empty is not the same as -f. I will look for another work around. Thank you 👍